2022-02-07 14:56:24,948 INFO  ==============================================================
2022-02-07 14:56:24,948 INFO  Id:    log_request_error_messages
2022-02-07 14:56:24,949 INFO  Dir:   log_request_end_to_end/
2022-02-07 14:56:24,949 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
2022-02-07 14:56:24,950 DEBUG Execution order hint: 0.0
2022-02-07 14:56:24,950 INFO  ==============================================================
2022-02-07 14:56:25,470 DEBUG EnvironmentC8y Setup
2022-02-07 14:56:25,471 INFO  EnvironmentC8y Setup
2022-02-07 14:56:25,472 DEBUG Process parameters for systemctl<serv_mapper1>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/serv_mapper1.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/serv_mapper1.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:25,477 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper1> (pid 21293)
2022-02-07 14:56:25,731 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-02-07 14:56:25,786 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 1 seconds
2022-02-07 14:56:25,787 INFO  Delaying execution by 10 seconds
2022-02-07 14:56:35,800 DEBUG Process parameters for sudo<tedge_connect_c8y>
  command line : /usr/bin/sudo /usr/bin/tedge connect c8y
    arg #1     : /usr/bin/tedge
    arg #2     : connect
    arg #3     : c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/tedge_connect_c8y.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/tedge_connect_c8y.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:35,809 DEBUG Waiting up to 600 secs for process sudo<tedge_connect_c8y> (pid 21299)
2022-02-07 14:56:47,544 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 11 secs
2022-02-07 14:56:47,546 DEBUG Process parameters for sudo<tedge_connect_c8y_test>
  command line : /usr/bin/sudo /usr/bin/tedge connect c8y --test
    arg #1     : /usr/bin/tedge
    arg #2     : connect
    arg #3     : c8y
    arg #4     : --test
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/tedge_connect_c8y_test.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/tedge_connect_c8y_test.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:47,551 DEBUG Waiting up to 600 secs for process sudo<tedge_connect_c8y_test> (pid 21458)
2022-02-07 14:56:47,858 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-02-07 14:56:47,860 DEBUG Process parameters for systemctl<serv_mosq2>
  command line : /usr/bin/systemctl status mosquitto
    arg #1     : status
    arg #2     : mosquitto
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/serv_mosq2.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/serv_mosq2.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:47,866 DEBUG Waiting up to 600 secs for process systemctl<serv_mosq2> (pid 21461)
2022-02-07 14:56:48,120 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-02-07 14:56:48,122 DEBUG Process parameters for systemctl<serv_mapper3>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/serv_mapper3.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/serv_mapper3.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:48,128 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper3> (pid 21469)
2022-02-07 14:56:48,383 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-02-07 14:56:48,385 DEBUG Process parameters for sudo<rm_logs>
  command line : /usr/bin/sudo rm -rf /tmp/sw_logs
    arg #1     : rm
    arg #2     : -rf
    arg #3     : /tmp/sw_logs
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/rm_logs.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/rm_logs.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:48,390 DEBUG Waiting up to 600 secs for process sudo<rm_logs> (pid 21470)
2022-02-07 14:56:48,645 INFO  Executed sudo<rm_logs>, exit status 0
2022-02-07 14:56:48,649 DEBUG Process parameters for sudo<move_logs>
  command line : /usr/bin/sudo sh -c 'mv /tmp/sw_logs/* /var/log/tedge/agent/'
    arg #1     : sh
    arg #2     : -c
    arg #3     : mv /tmp/sw_logs/* /var/log/tedge/agent/
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/move_logs.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/move_logs.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:48,655 DEBUG Waiting up to 600 secs for process sudo<move_logs> (pid 21472)
2022-02-07 14:56:48,912 INFO  Executed sudo<move_logs>, exit status 0
2022-02-07 14:56:48,912 DEBUG --- test execute
2022-02-07 14:56:49,079 WARN  caught <class 'SystemError'> while running test: field id is missing in response
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1209, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/run.py", line 39, in execute
    log_file_request_payload, self.project.deviceid)
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/environments/cumulocity.py", line 115, in trigger_log_request
    raise SystemError("field id is missing in response")
SystemError: field id is missing in response
2022-02-07 14:56:49,239 WARN  SystemError: field id is missing in response ... blocked 
2022-02-07 14:56:49,239 DEBUG --- test cleanup
2022-02-07 14:56:49,240 INFO  
2022-02-07 14:56:49,241 INFO  cleanup:
2022-02-07 14:56:49,241 DEBUG Running registered cleanup function: <bound method LogRequestVerifySearchTextError.cleanup_logs of <LogRequestVerifySearchTextError object at 0x753fbbd0>>
2022-02-07 14:56:49,242 DEBUG Process parameters for sudo<rm_logs>
  command line : /usr/bin/sudo sh -c 'rm -rf /var/log/tedge/agent/example-*'
    arg #1     : sh
    arg #2     : -c
    arg #3     : rm -rf /var/log/tedge/agent/example-*
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/rm_logs.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/rm_logs.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:49,248 DEBUG Waiting up to 600 secs for process sudo<rm_logs> (pid 21475)
2022-02-07 14:56:49,502 INFO  Executed sudo<rm_logs>, exit status 0
2022-02-07 14:56:49,504 DEBUG Process parameters for sudo<rm_tmp_logs>
  command line : /usr/bin/sudo rm -rf /tmp/sw_logs
    arg #1     : rm
    arg #2     : -rf
    arg #3     : /tmp/sw_logs
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/rm_tmp_logs.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/rm_tmp_logs.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:49,509 DEBUG Waiting up to 600 secs for process sudo<rm_tmp_logs> (pid 21478)
2022-02-07 14:56:49,763 INFO  Executed sudo<rm_tmp_logs>, exit status 0
2022-02-07 14:56:49,764 DEBUG Process parameters for sudo<send_failed>
  command line : /usr/bin/sudo /usr/bin/tedge mqtt pub c8y/s/us 502,c8y_LogfileRequest
    arg #1     : /usr/bin/tedge
    arg #2     : mqtt
    arg #3     : pub
    arg #4     : c8y/s/us
    arg #5     : 502,c8y_LogfileRequest
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/send_failed.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/send_failed.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:49,769 DEBUG Waiting up to 600 secs for process sudo<send_failed> (pid 21480)
2022-02-07 14:56:50,023 INFO  Executed sudo<send_failed>, exit status 0
2022-02-07 14:56:50,024 DEBUG Running registered cleanup function: <bound method EnvironmentC8y.myenvcleanup of <LogRequestVerifySearchTextError object at 0x753fbbd0>>
2022-02-07 14:56:50,024 DEBUG EnvironmentC8y Cleanup
2022-02-07 14:56:50,082 DEBUG Process parameters for sudo<tedge_disconnect_c8y>
  command line : /usr/bin/sudo /usr/bin/tedge disconnect c8y
    arg #1     : /usr/bin/tedge
    arg #2     : disconnect
    arg #3     : c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/tedge_disconnect_c8y.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/tedge_disconnect_c8y.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:50,087 DEBUG Waiting up to 600 secs for process sudo<tedge_disconnect_c8y> (pid 21486)
2022-02-07 14:56:53,365 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-02-07 14:56:53,366 DEBUG Process parameters for systemctl<serv_mapper5>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/serv_mapper5.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/Output/linux/serv_mapper5.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:56:53,371 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper5> (pid 21544)
2022-02-07 14:56:53,625 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-02-07 14:56:53,626 DEBUG ProcessUser cleanup function done.
2022-02-07 14:56:53,628 INFO  
2022-02-07 14:56:53,628 INFO  Test duration: 28.68 secs
2022-02-07 14:56:53,629 INFO  Test final outcome:  BLOCKED
2022-02-07 14:56:53,629 INFO  Test outcome reason: SystemError: field id is missing in response
2022-02-07 14:56:53,630 INFO  
